lib/wallaby/browser.ex

```diff
@@ -980,6 +980,63 @@ defmodule Wallaby.Browser do
     current_url(session) == "about:blank"
   end
 
+  @doc """
+  Accepts all subsequent JavaScript dialogs in the given session.
+  """
+  def accept_dialogs(%Session{}=session) do
+    Driver.accept_dialogs(session)
+    session
+  end
+
+  @doc """
+  Dismisses all subsequent JavaScript dialogs in the given session.
+  """
+  def dismiss_dialogs(%Session{}=session) do
+    Driver.dismiss_dialogs(session)
+    session
+  end
+
+  @doc """
+  Accepts one alert dialog, which is triggered within the specified `fun`.
+  Returns the last message that was presented to the user.
+  """
+  def accept_alert(%Session{}=session, fun) do
+    Driver.accept_alert(session, fun)
+  end
+
+  @doc """
+  Accepts one confirm dialog, which is triggered within the specified `fun`.
+  Returns the last message that was presented to the user.
+  """
+  def accept_confirm(%Session{}=session, fun) do
+    Driver.accept_confirm(session, fun)
+  end
+
+  @doc """
+  Dismisses one confirm dialog, which is triggered within the specified `fun`.
+  Returns the last message that was presented to the user.
+  """
+  def dismiss_confirm(%Session{}=session, fun) do
+    Driver.dismiss_confirm(session, fun)
+  end
+
+  @doc """
+  Accepts one confirm dialog, which is triggered within the specified `fun`.
+  Returns the last message that was presented to the user.
```

Comment:
Should mention that this is a prompt an the input value needs to be specified :)